test/runtests.jl

```diff
@@ -555,6 +555,36 @@ end
          0.01582155341041012, 0.029074956147127032, 0.023628812427424876])
 end
 
+@testset "Geometric LogLink" begin
+    # the default/canonical link is LogLink
+    gm22 = fit(GeneralizedLinearModel, @formula(Days ~ Eth + Sex + Age + Lrn), quine, Geometric())
+    test_show(gm22)
+    @test dof(gm22) == 8
+    @test deviance(gm22) ≈ 137.8781581814965
+    @test loglikelihood(gm22) ≈ -548.3711276642073
+    @test aic(gm22) ≈ 1112.7422553284146
+    @test aicc(gm22) ≈ 1113.7933502189255
+    @test bic(gm22) ≈ 1136.6111083020812
+    @test coef(gm22)[1:7] ≈ [2.8978546663153897, -0.5701067649409168, 0.08040181505082235, 
+                            -0.4497584898742737, 0.08622664933901254, 0.3558996662512287, 
+                            0.29016080736927813]
```

Comment:
Can you also test `stderror`?

```suggestion
    @test coef(gm22)[1:7] ≈ [2.8978546663153897, -0.5701067649409168, 0.08040181505082235, 
                            -0.4497584898742737, 0.08622664933901254, 0.3558996662512287, 
                             0.29016080736927813]
```

Reply:
Sure, I will update the test set accordingly.

Reply:
> 

Since it is a special case of `Negative Binomial` distribution with `r = 1`, we checked with `Negative Binomial`.